src/core/public/overlays/sidecar/helper.ts

```diff
@@ -44,3 +46,27 @@ export const getSidecarLeftNavStyle = (config: ISidecarConfig | undefined) => {
   }
   return {};
 };
+
+export const calculateNewPaddingSize = (
+  dockedMode: SIDECAR_DOCKED_MODE | undefined,
+  currentSize: number | undefined
```

Comment:
Nit: Based current implementation, the `currentSize` won't be `undefined` anymore. Can we remove the `| undefined` and the `currentSize` empty check below.